jenkins2/plugins.sh

```diff
@@ -0,0 +1,124 @@
+#! /bin/bash
+
+# Parse a support-core plugin -style txt file as specification for jenkins plugins to be installed
+# in the reference directory, so user can define a derived Docker image with just :
+#
+# FROM jenkins
+# COPY plugins.txt /plugins.txt
+# RUN /usr/local/bin/plugins.sh /plugins.txt
+#
+# Note: Plugins already installed are skipped
+#
+
+set -e
+
+echo "WARN: plugins.sh is deprecated, please switch to install-plugins.sh"
```

Comment:
It's like that in the upstream, but it supports both in the current build.
We can remove it if we like.